drivers/gpio/gpio_sam0.c

```diff
@@ -4,48 +4,132 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <errno.h>
-#include <kernel.h>
-#include <device.h>
-#include <init.h>
-#include <soc.h>
 #include <gpio.h>
+#include <soc.h>
+
+#include "gpio_utils.h"
 
 struct gpio_sam0_config {
 	PortGroup *regs;
 };
 
-#define DEV_CFG(dev) \
+struct gpio_sam0_data {
+	sys_slist_t cb;
+	u32_t cb_pins;
+};
+
+DEVICE_DECLARE(gpio_sam0_0);
+
+#define DEV_CFG(dev)							     \
```

Comment:
In Zephyr we tend to not align \ at column 80 but rather place it directly (one space distance) after the last character on the line, just as it was before the change.

Reply:
Ack.  I'm using clang-format for style and will update the defaults & fix.

Reply:
Done.